x-pack/plugins/security/public/management/roles/model/secured_sub_feature.ts

```diff
@@ -14,6 +14,7 @@ import { SubFeaturePrivilegeGroup } from './sub_feature_privilege_group';
 export class SecuredSubFeature extends SubFeature {
   public readonly privileges: SubFeaturePrivilege[];
   public readonly privilegesTooltip: string;
+  public readonly description: string;
```

Comment:
__question:__ is there any reason you don't want to add `description()` getter to the parent `SubFeature` class like you did in `KibanaFeature` (just for the consistency sake)?

Reply:
added the getter for the subfeature description

Reply:
Sorry, I meant that you added `description` getter to `KibanaFeature` that just re-exposes value from `KibanaFeatureConfig`, and since you added `description` to `SubFeatureConfig` as well it makes sense to mirror the same getter in `SubFeature` (in features plugin, not security plugin, like it's done with `name` for example). This way we won't need any changes in `SecuredSubFeature`.

Does that make sense?

Reply:
ahh i see what you mean now!